### DISCUSSION TITLE
Magnitude's problem

### DISCUSSION THREAD

**Discussion by meliastro** [Stellarium/stellarium#2776] [Q&A] [answered]
Hi,
I would like to know  why I obtain 2 differents magnitude for the same objet. With a Macbook Air, I obtain 6,90 for the object M30 and with a PC (windows) I obtain 7,70.

Thanks

![Capture_decran_le_2022-10-05_a_17 09 17_LI](https://user-images.githubusercontent.com/116565634/197526165-33bf652b-72e9-4676-a4bd-02c7a20557a6.jpg)
![stellarium-010_LI](https://user-images.githubusercontent.com/116565634/197526382-0c13c43a-c7bd-49c7-ae87-99c23cbeed91.jpg)

**Comment by gzotti** [comment]
Same program versions?

**Comment by meliastro** [comment]
version 0.20.4 (windows) and version 0.22.2 (Macbook Air)

**Reply by gzotti** [reply, marked as answer]
Now can you see the answer? If not, install 1.0 on both computers and try again.
